### ISSUE TITLE
Add division capability.

### ISSUE THREAD

**Issue by ZackSheffieldSEL** [Schweitzer-Engineering-Laboratories/fixedpoint#2]
As of 1.0.0, the `__truediv__` and `__floordiv__` operators return `NotImplemented`. 

Here are some resources for implemented fixed point division:

* [Viet's Blog](https://blog.veitheller.de/Fixed_Point_Division.html)
* Randy Yate's [Fixed Point Airthmetic: An Introduction](https://github.com/Schweitzer-Engineering-Laboratories/fixedpoint/files/4523598/fp.pdf)

**Comment by heteloo** [comment]
You can use the division algorithm simply by making it a function.
(I'm a beginner, so I don't know if this is a good way.)

**Comment by zks0002** [comment]
@heteloo, you are correct, but the idea of division for fixed point arithmetic is somewhat ambiguous. The 2 links in my first comment address ways to implement division, but given that fixed-point division is a somewhat niche and specialized operation I think it probably makes sense to leave the division operation undefined and let users implement it themselves if needed.
